docs/sources/storage/_index.md

```diff
@@ -50,9 +50,17 @@ You may use any substitutable services, such as those that implement the S3 API
 
 ### Single-Store
 
+Single-Store refers to the using object storage as the storage medium for both Loki's index as well as its data ("chunks"). There are two supported modes:
+
+#### Suggested (2.8+): tsdb
+
+The [TSDB index store]({{< relref "../operations/storage/tsdb" >}}) improves query performance, reduces TCO and has the same feature parity as "boltdb-shipper".
+
+#### Deprecated (2.0+): boltdb-shipper
+
```

Comment:
Available in Loki 2.0 through 2.7, BoldDB (also known as `boltdb-shipper`) has been deprecated and will be removed in a future release.  This single store configurations for Loki utilizes the chunk store for both chunks and the index, requiring just one store to run Loki.